rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/telemetry/TbMsgDeleteAttributes.java

```diff
@@ -67,7 +70,11 @@ public void onMsg(TbContext ctx, TbMsg msg) throws ExecutionException, Interrupt
         if (keysToDelete.isEmpty()) {
             ctx.tellSuccess(msg);
         } else {
-            ctx.getTelemetryService().deleteAndNotify(ctx.getTenantId(), msg.getOriginator(), scope, keysToDelete, new AttributesDeleteNodeCallback(ctx, msg, scope, keysToDelete));
+            FutureCallback<Void> callback = new TelemetryNodeCallback(ctx, msg);
```

Comment:
please create this callback only if sendAttributesDeletedNotification is set to false